.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: NodeJS with Webpack
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    # Consider setting up a dependency cache:
```

Comment:
might as well add "TODO" to this comment so it's more easily searchable

Reply:
I decided to remove it altogether in favour of filing Issue #61 .